initializers/prefixes.yml

```diff
@@ -0,0 +1,21 @@
+#- description: prefix1
+#  prefix: 10.1.1.0/24
+#  site: AMS 1
+#  status: Active
+#  tenant: tenant1
+#  vlan: vlan1
+#- description: prefix2
+#  prefix: 10.1.2.0/24
+#  site: AMS 2
+#  status: Active
+#  tenant: tenant2
+#  vlan: vlan2
+#  is_pool: true
+#  vrf: vrf2
+#  tenant: tenant2
```

Comment:
The tenant is defined twice. Duplicate keys are not allowed by the Netbox code.

```suggestion
```

Reply:
Fixed.